README.md

```diff
@@ -100,6 +100,38 @@ ssh, scp, rsync:
 - drush 6.3.0
 - jQuery 1.10.2
 
+## Run in a multiple VM configuration
+To run this vm side by side with Islandora 8 or another VM with conflicting ports do the following
+
+Prerequisite vagrant plugins for full operation.
+```shell
+  # For more info https://github.com/dotless-de/vagrant-vbguest
+$ vagrant plugin install vagrant-vbguest
+
+  # For more info https://github.com/cogitatio/vagrant-hostsupdater
+$ vagrant plugin install vagrant-hostsupdater
+```
+To start this as a second VM either set a enviroment variable `ISLANDORA_VAGRANT_MULTIPLE_ISLANDORAS` to TRUE or run the following command.
+```shell
+$ ISLANDORA_VAGRANT_MULTIPLE_ISLANDORAS='TRUE' vagrant up
+```
+
+#### NOTE:
+`vagrant-hostsupdater` will temporarily update the host machine's `/etc/hosts` file and remove it when vagrant is destroyed.
```

Comment:
Add: "Also note that each vm must be `halt`ed separately, from within the directories on the host where they were started with `vagrant up`."

Reply:
Probably should wrap vagrant up in backticks since it's a command: `vagrant up`.

Reply:
I think I got them all.